app/views/transactions/index.html.erb

```diff
@@ -0,0 +1,39 @@
+<div class="flex h-full flex-col overflow-y-scroll bg-white shadow-xl  max-w-screen-lg px-4 py-2 mx-auto">
+  <div class="flex-1 overflow-y-auto py-2 px-4 sm:px-6">
+    <div class="flex items-start justify-between">
+      <h2 class="text-lg font-medium text-gray-900" id="slide-over-title">Total Amount for group <%= @group.name %>: <%= @transactions_sum || 0 %> $</h2>
+    </div>
+
+    <div class="mt-8">
+      <div class="flow-root">
+        <ul role="list" class="-my-6 divide-y divide-gray-200">
+          <% @transactions.each do |transaction| %>
+            <li class="flex py-6">
+
+              <div class="ml-4 flex flex-1 flex-col">
+                <div>
+                  <div class="flex justify-between text-base font-medium text-gray-900">
+                    <div>
+                      <h3><%= transaction.name %> </h3>
+                      <span class="text-gray-500 text-sm">created on <%= transaction.created_at.strftime("%B %e, %Y")%> </span>
+                    </div>
+                    <p class="ml-4"><%= transaction.amount %> $</p>
+                  </div>
+                </div>
+              </div>
+
+            </li>
+          <% end %>
+        </ul>
+      </div>
+    </div>
+    <div class="py-6 px-4 sm:px-6">
+      <div class="mt-6">
+        <%= link_to 'Create transaction', new_group_transaction_path(group_id: @group.id), class: "flex items-center justify-center border border-transparent bg-secondColor px-6 py-3 text-base font-medium text-white shadow-sm" %>
```

Comment:
- [ **OPTIONAL** ] I know this was not mentioned by the previous reviewer. But according to the project requirements it was stated that:  **When a user clicks (or taps) on a category item(that you use as a group item in your app), the application navigates to the transactions page for that category. That means you don't need to create a `category` separately as your `Group` page was supposed to be for `Categories` that why I suggested to remove that functionality. Right now it seems as a repetition**.
Right now now your app behavior like this: before creating a new transaction you have to create again another category that you should have once clicked from categories list.

If you take a look on the  image below, you should use the clicked category item from the the home page so its `icon` at the top, `name`,...
![Screenshot from 2023-01-14 16-27-05](https://user-images.githubusercontent.com/83042742/212477074-3c84a615-7781-42ea-b238-589abf4b9d68.png)